WcaOnRails/app/webpacker/components/DelegateProbations/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Button, Input, Table } from 'semantic-ui-react';
+import UserBadge from '../UserBadge';
+import { startProbationUrl, endProbationUrl } from '../../lib/requests/routes.js.erb';
+import { fetchWithAuthenticityToken } from '../../lib/requests/fetchWithAuthenticityToken';
+
+export default function DelegateProbations({ probationRoles, probationUsers }) {
+  const [wcaId, setWcaId] = React.useState('');
+
+  function startProbation() {
+    fetchWithAuthenticityToken(startProbationUrl, {
+      method: 'POST',
+      body: JSON.stringify({ wcaId }),
+      headers: { 'Content-Type': 'application/json' },
+    }).then(async () => {
+      window.location.reload();
```

Comment:
This likely goes along with Gregor's mention of how component should `get` data but it'll be real nice when that's implemented, this should just refetch a specific request as opposed to making a full page refresh (poor UX)